crates/editor/src/editor_settings.rs

```diff
@@ -12,7 +12,7 @@ pub struct EditorSettings {
     pub hover_popover_enabled: bool,
     pub show_completions_on_input: bool,
     pub show_completion_documentation: bool,
-    pub completion_documentation_secondary_query_debounce: u64,
+    pub completion_documentation_secondary_query_debounce: Option<u64>,
```

Comment:
NIT: I think we can silently ignore that and remove entirely, as we did with a few other deprecated items, stating that in the release notes.

Reply:
Well that explains why I don't see any other deprecated settings!  Thanks